Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM golang:1.14.4-alpine3.12 AS build
+FROM golang:1.14-alpine AS build
```

Comment:
1. I don't think we care which version of alpine built the binary
2. We can afford not specifying patch version, if we believe in semver